js/script.js

```diff
@@ -0,0 +1,43 @@
+$(function(){
+  var carouselList = $("#images");
+  var interval = setInterval(changeSlide, 3000);
+
+  function moveSlide(){
+    var firstItem = carouselList.find("li:first");
+    var lastItem = carouselList.find("li:last");
+    lastItem.after(firstItem);
+    carouselList.css({marginLeft:0});
+  }
+
+  function carouselStart(){
+    interval;
+  }
+
+  function changeSlide(){
+    carouselList.animate({marginLeft:-500}, 3000, moveSlide);
+  }
+
+  function moveSlidePrev(){
+    var firstItem = carouselList.find("li:first");
+    var lastItem = carouselList.find("li:last");
+    firstItem.before(lastItem);
+    carouselList.css({marginLeft:0});
+  }
+
+  $('#left').on('click', function(){
+    clearInterval(interval);
+    carouselList.animate({marginLeft:500}, 1000, moveSlidePrev);
+    setInterval(changeSlide, 3000);
```

Comment:
setInterval return an  id - when you clean your interval - it's ok, but for current moment

Reply:
I cant't understand what you mean? What id? I commented out this line with setInterval.

Reply:
every function return something, setInterval returns an id of interval. your new setInterval should be assign to previous variable